cfn-templates/cid-cfn.yml

```diff
@@ -4,67 +4,99 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: 'Prerequisites'
+          default: 'Common Parameters'
         Parameters:
           - PrerequisitesQuickSight
-          - PrerequisitesCUR
           - PrerequisitesQuickSightPermissions
-      - Label:
-          default: 'Mandatory parameters'
-        Parameters:
           - QuickSightUser
-          - CURBucketPath
-          - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: CUDOS Cost-Intelligence-Dashboard and KPI-Dashboard. Require deployment of CUR via CloudFormation (cur-aggregation.yaml) or manually 24h before.
         Parameters:
+          - CURBucketPath
           - DeployCUDOSDashboard
           - DeployCostIntelligenceDashboard
           - DeployKPIDashboard
+      - Label:
+          default: Trusted Advisor and Compute Optimiser Dashboards. Require deployment of Optimization Data Collection Lab (https://wellarchitectedlabs.com/cost/300_labs/300_optimization_data_collection/)
+        Parameters:
+          - OptimizationDataCollectionBucketPath
           - DeployTAODashboard
           - DeployComputeOptimizerDashboard
-          - CostOptimizationDataCollectionBucketPath
       - Label:
-          default: 'Optional Parameters (do not change unless you know why)'
+          default: 'Technical Parameters. Please do not change.'
         Parameters:
           - AthenaWorkgroup
           - AthenaQueryResultsBucket
-          - CURDatabaseName
+          - DatabaseName
           - CURTableName
           - CidVersion
           - Suffix
           - QuickSightDataSetRefreshSchedule
           - LambdaLayerBucketPrefix
+    ParameterLabels: 
+      PrerequisitesQuickSight:
+        default: "I have QuickSight Enterprise Edition AND i have a SPICE capacity in the current region."
+      PrerequisitesQuickSightPermissions:
+        default: "I understand that i need to set up manually Permission of QuickSight to access CUR backet and Query results bucket, and then relaunch dataset refresh manually the first time."
```

Comment:
Suggested version: I have set up manually Permission of QuickSight to access CUR bucket and Athena query results bucket.
'and then relaunch dataset refresh manually the first time' - is it because this cfn doesn't set up dataset refresh?